.github/workflows/velox_be.yml

```diff
@@ -94,18 +94,18 @@ jobs:
       - name: Copy golden files from container to host
         if: failure()
         run: |
-          $PATH_TO_GLUTEN_TE/$OS_IMAGE_NAME/gha/gha-checkout/cp.sh /tmp/tpch-approved-plan/ /tmp/$GITHUB_RUN_ID/tpch-approved-plan
+          $PATH_TO_GLUTEN_TE/$OS_IMAGE_NAME/gha/gha-checkout/cp.sh /tmp/tpch-approved-plan/ /tmp/$GITHUB_RUN_ID/spark322/tpch-approved-plan
       - name: Upload golden files
         if: failure()
         uses: actions/upload-artifact@v4
         with:
           name: golden-files-spark32
           path: |
-            /tmp/${{ github.run_id }}/tpch-approved-plan/**
+            /tmp/${{ github.run_id }}/spark322/tpch-approved-plan/**
```

Comment:
does the path correct ?

Reply:
Yes, this path is CI Local Disk Path, which will be created when copy golden files from docker image.